Assets/LeapMotionModules/UI/Scripts/BoxDepenetrator.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BoxDepenetrator : MonoBehaviour {
+
+  public SphereCollider sphere;
+
+  [Header("Automatic if null (this)")]
+  public BoxCollider box;
+
+  public struct DepenetrationRay {
+    public Vector3 position;
+    public Vector3 direction;
+  }
+  
+  private DepenetrationRay _depenetrationRay;
+  private bool _requiresDepenetrationThisFrame = false;
+
+  void Start() {
+    if (box == null) box = GetComponent<BoxCollider>();
```

Comment:
You can use the AutoFind attribute in order to automatically fill the box field with sibling boxColliders.